README.md

```diff
@@ -447,10 +447,17 @@ This repository contains a curated list of awesome open source libraries that wi
 
 
 ## Industry Strength CV
+* [Albumentations](https://github.com/albumentations-team/albumentations) ![](https://img.shields.io/github/stars/albumentations-team/albumentations.svg?style=social) - Albumentations is a Python library for image augmentation.
 * [Deep Lake](https://github.com/activeloopai/deeplake) ![](https://img.shields.io/github/stars/activeloopai/deeplake.svg?style=social) - Deep Lake is a data infrastructure optimized for computer vision.
+* [Diffusers](https://github.com/huggingface/diffusers) ![](https://img.shields.io/github/stars/huggingface/diffusers.svg?style=social) - Diffusers provides pretrained diffusion models across multiple modalities, such as vision and audio, and serves as a modular toolbox for inference and training of diffusion models.
+* [FiftyOne](https://github.com/voxel51/fiftyone) ![](https://img.shields.io/github/stars/voxel51/fiftyone.svg?style=social) - FiftyOne is an open-source tool for building high-quality datasets and computer vision models
 * [iGibson](https://github.com/StanfordVL/iGibson) ![](https://img.shields.io/github/stars/StanfordVL/iGibson.svg?style=social) - iGibson is a simulation environment providing fast visual rendering and physics simulation based on Bullet.
+* [Lightly](https://github.com/lightly-ai/lightly) ![](https://img.shields.io/github/stars/lightly-ai/lightly.svg?style=social) - Lightly is a computer vision framework for self-supervised learning.
 * [KerasCV](https://github.com/keras-team/keras-cv) ![](https://img.shields.io/github/stars/keras-team/keras-cv.svg?style=social) - KerasCV is a library of modular computer vision oriented Keras components.
+* [Kornia](https://github.com/kornia/kornia) ![](https://img.shields.io/github/stars/kornia/kornia.svg?style=social) - Kornia is a differentiable computer vision library for PyTorch.
+* [OpenCV](https://github.com/opencv/opencv) ![](https://img.shields.io/github/stars/opencv/opencv.svg?style=social) - OpenCV is a library of programming functions mainly for real-time computer vision.
```

Comment:
This would be only one to remove, openCV is great but falls under the lower-level tooling (like NLTK, sklearn, etc)

Reply:
I do not see any reason from CONTRIBUTE.md that what it means to be excluded as a production tool. The standard seems so versatile for each tool, OpenCV contains a lot of high-level ML algorithms as well. Have you tried them before? @axsaucedo

Reply:
I have indeed used opencv and as mention is a great library but again same point as above - the rest of PR looks good, I am keen to avoid a long discussion about this tool in favour of merging the pr so let's remove so we can merge

Reply:
While it is not a good practice to make the line blur, as an OSS contributor, we need to know the standard or this extra untransparency indeed makes me frustrated and this disobeys the aim of OSS. @axsaucedo

Reply:
I appreciate your frustration but as mentioned in #392 we can't discuss forever on opinions, and at this stage having had this discussion a few times already keen to remove so we can merge

Reply:
I agree with you we should not discuss a matter forever. In order to prevent the same thing from happening in the future, I desire to have much clearer guidelines in the commitment, so I do not have to spend time collecting the unneeded tools and you do not bother to review my unneeded tools. @axsaucedo